src/app/views/app-sections/AppTitle.tsx

```diff
@@ -9,12 +9,16 @@ export function appTitleDisplayOnFullScreen(
     ): React.ReactNode {
 
     return <div style={{ display: 'flex', width: '100%' }}>
-      <IconButton
-        iconProps={{ iconName: 'GlobalNavButton' }}
-        className={classes.sidebarToggle}
-        title='Minimise sidebar'
-        ariaLabel='Minimise sidebar'
-        onClick={() => toggleSidebar()} />
+      <TooltipHost
+        content='Minimise sidebar'
```

Comment:
misspelt.  should be z

Reply:
also what about localization here?

Reply:
> misspelt. should be z

@ddyett depends on which english pronunciation you're using :alien:

Reply:
all the more reason for localization.  since the most popular page is en-US probably should be z.